rpa-builder/6.4/modules/ROOT/pages/common-properties.adoc

```diff
@@ -45,12 +45,12 @@ A name for the timer. RPA Builder automatically adds this property in the Ping a
 ** *Script Feedback / Reporting Function*
 *** *Record Timer*
 +
-When this option is enabled, the runtime measured for the Ping and Service Check Action Steps is passed to RPA Manager via a linked Data Source.
+When this option is enabled, the execution time measured for the Ping and Service Check Action Steps is passed to RPA Manager.
```

Comment:
"When this option is enabled, the execution time measured for the Ping and Service Check Action Steps is recorded and output in the Time outbound variable of the action steps"
Something like this, with out the RPA Manager, no measurement data is passed to RPA Manager, yet.